Modulefile

```diff
@@ -1,4 +1,4 @@
-name    'hardening/nginx_hardening'
+name    'hardening-nginx_hardening'
```

Comment:
that looks strange? We used the same hardening/* for ssh and os. Any specific reason?